controllers/apartments.rb

```diff
@@ -8,7 +9,18 @@
   erb :"apartments/new"
 end
 
+post '/apartments/new' do
```

Comment:
should post to `/apartments`. `/apartments/new` is for displaying the form to create an apartment.